.travis.yml

```diff
@@ -20,6 +20,8 @@ dist: precise # downgrade to avoid jdk7 SSL Bug https://github.com/docker-librar
 jdk:
   - oraclejdk7 # openjdk7 has SSL Bug avoid jdk7 SSL Bug https://github.com/docker-library/openjdk/issues/117
   - oraclejdk8
+  - openjdk9
```

Comment:
Java 9 is already eol, the build on travis was green, so I'd say we drop it

Reply:
Done.